docs/en/observability/view-observability-alerts.asciidoc

```diff
@@ -59,20 +59,26 @@ and an alert has "Recovered" when that condition, which previously matched, is c
 [[view--alert-details]]
 == View alert details
 
-When you have searched and filtered for a specific alert, you may want to inspect the alert details.
-From the Alerts table, select image:images/flyout-icon.png[View details] to view additional information summarizing the alert.
+There are a few ways to inspect the details for a specific alert.
 
-You can find the current status of the alert, along with its duration, and when it was last
-updated. To help you determine what caused the alert, you can view the expected and actual
-threshold values, and the rule that produced the alert.
+From the Alerts table, you can select image:images/flyout-icon.png[Diagonal line with arrows icon used to open the "View details" flyout,height=22] to open the alert detail flyout to view more information summarizing the alert without leaving the page.
+There you'll see the current status of the alert, its duration, and when it was last updated.
+To help you determine what caused the alert, you can view the expected and actual threshold values, and the rule that produced the alert.
 
-To view the alert in the specific app it was initially created from, select image:images/app-link-icon.png[Link to app icon]. 
+[role="screenshot"]
+image::view-alert-details.png[View alert details flyout on the Alerts page]
+
+You can further inspect the rule from the alert detail flyout by clicking *View rule details* or
```

Comment:
I'd break this down to make it easier to scan and maybe avoid saying "you can" because it gets a little repetitive. Maybe something like:

```
To further inspect the rule:
* From the alert detail flyout, click *View rule details*.
* From the Alerts table, use the image:images/action-dropdown.png[Three dots used to expand the "More actions" menu,height=22] and click *View rule details*.
```